python/test/test_polars.py

```diff
@@ -42,6 +42,13 @@ def example_lf(margin=None, **kwargs):
     return lf_domain, lf
 
 
+def test_expr_domain():
+    lf_domain, _ = example_lf()
+    dp.expr_domain(lf_domain, context="filter")
```

Comment:
I don't see "filter" anywhere else in this PR, so what is special about it? Are there other values we could check?

Reply:
Removed based on changes in the ExprDomain PR.